target/linux/ramips/image/mt7620.mk

```diff
@@ -36,6 +36,14 @@ define Device/alfa-network_ac1200rm
 endef
 TARGET_DEVICES += alfa-network_ac1200rm
 
+define Device/apac1200
+  DTS := APAC1200
+  DEVICE_TITLE := V7 APAC1200
+  DEVICE_PACKAGES := kmod-ath10k kmod-ath10k-firmware-qca988x
```

Comment:
kmod-ath10k-firmware-qca988x <- wrong package name. Should be ath10k-firmware-qca988x